src/codeu/chat/util/Tokenizer.java

```diff
@@ -0,0 +1,62 @@
+private StringBuilder token;
```

Comment:
This and most everything below it should be inside the class definition (at the very end of this file) as member variables and methods.